src/main/java/game/SoccerGameManager.java

```diff
@@ -0,0 +1,74 @@
+package game;
+
+import entity.Attacker;
+import entity.GoalKeeper;
+import entity.Midfielder;
+import entity.Striker;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Collectors;
+
+public class SoccerGameManager {
+    Scanner sc = new Scanner(System.in);
+    private List<Attacker> attackers = new ArrayList<>();
+    private GoalKeeper goalKeeper;
+    private Integer matchpoint = 0;
+    private void setStrikers() {
+        System.out.print("공격수의 이름을 입력하세요: ");
+        List<String> strikers = Arrays.stream(sc.nextLine().split(", "))
+                .collect(Collectors.toCollection(ArrayList::new));
+
+        for(String player : strikers){
+            attackers.add(new Striker(player));
+        }
```

Comment:
이런식으로 스트림을 사용한다면 더 깔금한 코드가 될 수 있겠네요

```suggestion
        System.out.print("공격수의 이름을 입력하세요: ");
        List<String> strikers = Arrays.stream(sc.nextLine().split(", "))
                .forEach(attacker -> attackers.add(new Striker(attacker));

```